javascript/operations.js

```diff
@@ -462,7 +476,7 @@ export default {
     before(document, operation)
     operate(operation, () => {
       const { data, columns } = operation
-      console.table(data, columns || [])
+      console.table(data, columns != null ? columns : [])
```

Comment:
Irrelevant, but TIL decimal values are converted to a string, when using `to_json`

Reply:
That strikes me as super WTF but outside of our scope or ability to fix. Right?